internal/venice-common/src/main/java/com/linkedin/venice/meta/SystemStore.java

```diff
@@ -217,35 +217,31 @@ public void setPartitionerConfig(PartitionerConfig value) {
   }
 
   /**
-   * This function is used to check the precondition for store deletion.
-   * Since the system store should deleted together with the corresponding regular Venice store, so
-   * this function will delegate the call to the regular Venice store.
-   * @return
+   * System stores are internal stores, and even the corresponding user store is not writable, and the
+   * internal system stores should be writable, which is controlled by the {@link #zkSharedStore}.
```

Comment:
Nit points.

"and the internal" -> "the internal"
"should be writable" -> "could be writable"?